stories/Input/SelectStory.tsx

```diff
@@ -38,17 +38,15 @@ const props = {
       defaultValue: '',
       possibleValue: 'any',
       require: 'no',
-      description:
-        'Sets error state on component. If string, the error will be shown below the select',
+      description: `WARNING: Will only allow string type in the future, don't customize error message style. Sets error state on component. If string, the error will be shown below the select`,
     },
     {
       name: 'renderError',
```

Comment:
Seems to neither `employers` nor `dst` use `renderError`. Do you think we can just remove it in this PR?

Reply:
no, I am only fix UI mismatching